aws-ssm-patchbaseline/inputs/inputs_1_invalid.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
please put a correct format of json file as inputs_1_invalid.json content, meaning: the format is still valid, but the input values are not valid.